src/stylesheets/elements/_table.scss

```diff
@@ -5,3 +5,34 @@
 .usa-table--borderless {
   @extend %usa-table--borderless;
 }
+
+.usa-table--compact {
```

Comment:
Is this the same as mixin from [`mixins/_usa-table-styles.scss`](https://github.com/uswds/uswds/pull/3825/files?file-filters%5B%5D=.scss&file-filters%5B%5D=.yml#diff-d275d7200f7b6e40c0c972f9a29a40b8ce9eace6110bcd47dd06addc5b67c4b4)?

Reply:
It is. The first variant -- borderless -- was set up this way, so I assumed it was a convention. If you (or anyone else) thinks this should be different, I'd love to hear more.